en/lessons/advanced/erlang.md

```diff
@@ -34,8 +34,8 @@ For a complete list of the modules available, see the [Erlang Reference Manual](
 
 ## Erlang Packages
 
-In a prior lesson we covered Mix and managing our dependencies.
-Including Erlang libraries works the same way.
+In a prior lesson, we covered Mix and managing our dependencies.
+Including Erlang, libraries work the same way.
```

Comment:
I think the edits on line 38 are not necessary. We are trying to say here that the process of including Erlang libraries into your Elixir project works the same way as does using Mix to manage your Elixir app's dependencies. 

If you think there is a clearer way to state this, let us know! But the original sentence is grammatically correct as far as I understand it.